mdbxdist/VERSION.txt

```diff
@@ -1 +1 @@
-0.13.1.56
+0.12.13.0
```

Comment:
Why is the version going backward?
Should this read `0.13.5`?

Reply:
This is an obsolete file from the old version, the new version provides `VERSION.json`. The update to the new version was not done quite correctly, I will try to fix it soon.